app/views/posts/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>the given users posts list </h1>
\ No newline at end of file
+<h1>Here is the list of posts:</h1>
+<div>
+<h2><%= @user.name %></h2>
+<p>Number of posts: <%=@user.PostsCounter.nil? ? 0 :@user.user_posts %></p>
+  <% @posts.each do |post| %>
+    <div class='post-card'>
+      <p><%= post.text %></p>
+      <div class='likes-comments'>

```

Comment:
- [ ] Following the project requirements, making your pages look like the provided wireframes would be nicer.